sports/models.py

```diff
@@ -7,23 +7,54 @@ class Participant(Person):
     date_of_birth = models.DateField(_('date of birth'))
     wheelchair_bound = models.BooleanField(_('wheelchair bound'), default=False)
     sport = models.ForeignKey('Sport')
-    sport_details = models.ManyToManyField('SportDetail')
-    team = models.ManyToManyField('Team')
+    sport_details = models.ManyToManyField('SportDetail', blank=True)
```

Comment:
Hij hoeft wat mij betreft niet verplicht te worden bij het invullen van het formulier (hence, `blank=True`) en een null=True heeft geen zin bij een M2M relationship toch? Omdat er geen echte tabel is die NULL kan zijn?